server/src/com/cloud/server/ConfigurationServerImpl.java

```diff
@@ -894,6 +894,13 @@ protected void injectSshKeysIntoSystemVmIsoPatch(String publicKeyPath, String pr
         } else {
             command = new Script("/bin/bash", s_logger);
         }
+        String osname = System.getProperty("os.name");
+        if (osname.startsWith("Windows")) {
+            scriptPath = scriptPath.replaceAll("\\\\" ,"/" );
```

Comment:
testWindowsScript#ConfigurationServerImplTest TestCase might break with this change.